Exiled.API/Features/Lift.cs

```diff
@@ -75,8 +75,14 @@ internal Lift(ElevatorChamber elevator)
         /// <summary>
         /// Gets a value of the internal doors list.
         /// </summary>
+        [Obsolete("Use ElevatorDoors property instead.")]
         public IReadOnlyCollection<ElevatorDoor> Doors => internalDoorsList;
 
+        /// <summary>
+        /// Gets a value of the internal doors list.
+        /// </summary>
+        public IReadOnlyCollection<Doors.ElevatorDoor> ElevatorDoors => internalDoorsList.Select(x => Door.Get(x).As<Doors.ElevatorDoor>()).ToList();
```

Comment:
I feel like the name `ElevatorDoors` is redundant and `Doors` is better in this case.

Reply:
Can I just remove doors property? It's a breaking change but we gonna have major release

Reply:
you can make breaking change yeah same for your Door API